aff4/aff4_image.cc

```diff
@@ -359,6 +359,9 @@ AFF4Status AFF4Image::LoadFromURN() {
         }
     }
 
+    // By default we cache 32 MiB of bevys
+    chunk_cache_size = (32 * 1024 * 1024) / chunk_size;
```

Comment:
Should this be configurable? Some applications will not benefit from the cache at all or might want to restrict memory usage.

Reply:
Is is configurable.  `chunk_cache_size` is a public member and the user can set it to whatever they like and the cache will respond accordingly

Reply:
We probably need some way to configure general things about the library. Just having this as a public member  requires code to open the stream, then set the attribute. Often you dont even know if it is an aff4_image because you open it using the stream interface so you can not set the caching attribute.

I am thinking of putting these in the resolver as part of the volatile namespace. This way if users dont want to use cache they can just set the cache size to 0 or whatever by setting an attribute on the configure object.

So maybe here we could do:
if (resolver->Get(AFF4_LIBRARY_CONFIG, AFF4_CONFIG_CACHE_SIZE, value) == STATUS_OK) {
      chunk_cache_size = value.value / chunk_size;
}

Where we can add those constants to lexicon.inc.
Happy to leave it now but I think we need to add a new feature to configure the library.